.flake8

```diff
@@ -1,6 +1,11 @@
 [flake8]
 
 filename = ./src/*
+
+exclude =
+    *.html
+    __pycache__
```

Comment:
I didn't know it can be without commas :D

Anyway I think we need to add both commas here to be consistent with another lists in this file